src/routers/api.ts

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import { ROUTES } from '../utils/constants';
+import userRouter from './user';
+import mailingListRouter from './mailingList';
+
+const router = express.Router();
+
+router.use(ROUTES.USER, userRouter);
+router.use(ROUTES.MAILING_LIST, mailingListRouter);
```

Comment:
Route definitions are less useful because they are only used here rn

Reply:
Loving it